cmake/GinkgoConfig.cmake.in

```diff
@@ -81,6 +82,8 @@ set(GINKGO_DPCPP_FLAGS @GINKGO_DPCPP_FLAGS@)
 set(GINKGO_MKL_ROOT @GINKGO_MKL_ROOT@)
 set(GINKGO_DPL_ROOT @GINKGO_DPL_ROOT@)
 
+set(GINKGO_HAVE_MPI @GINKGO_HAVE_MPI@)
```

Comment:
You may have misunderstood my comment slightly - I was suggesting keeing GINKGO_HAVE_CUDA_AWARE_MPI, but removing the other two, since GINKGO_BUILD_MPI and GINKGO_HAVE_MPI are effectively the same thing.

Reply:
But I think we need to have either `GINKGO_BUILD_MPI` or `GINKGO_HAVE_MPI`, right ?

Reply:
yes, one of both. I would prefer BUILD_...
So overall, we need BUILD_MPI and HAVE_CUDA_AWARE_MPI